src/style.css

```diff
@@ -1 +1,77 @@
-/* Styles go here */
+body {
+  margin: 0;
+  --color: #00acdc;
+}
+
+.menu {
+  padding: 0 50px;
+}
+
+.menu__container {
+  max-width: 1200px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0 auto;
+}
+@media (max-width: 1024px) {
+  .menu__container {
+    max-width: 1024px;
+  }
+}
+
+.menu__logo {
+  display: inline-block;
+  width: 40px;
+  height: 40px;
+  padding: 10px 0;
+}
+
+.nav__list {
+  list-style-type: none;
+  display: flex;
+  align-items: center;
+  padding: 0;
+  margin: 0;
+}
+
+.nav__item {
+  margin-right: 20px;
+}
+
+.nav__item:last-child {
+  margin-right: 0;
+}
+
+.nav__link {
+  display: flex;
+  align-items: center;
+  height: 60px;
+  position: relative;
+  font-family: "Roboto", sans-serif;
```

Comment:
It's better to initialize the font-family in the selector because usually the site uses one font and it should be available for all elements. Also better to add more fallback fonts accordingly to paragraph 3 in checklist